app-frontend/src/app/components/selectedScenesModal/selectedScenesModal.controller.js

```diff
@@ -23,7 +23,6 @@ export default class SelectedScenesModalController {
         let sceneIds = Array.from(this.selectedScenes.keys());
         this.projectService.addScenes(this.resolve.project.id, sceneIds).then(
             () => {
-                this.resolve.scenes.clear();
                 this.close({ $value: sceneIds });
             },
             (err) => {
```

Comment:
Not related to this PR but the next line down fails if you hit an error condition because the constructor doesn't set `this.$log = $log`; that'd be nice to fix along with this.